test/odbc/psqlodbc/test_pg_varchar.cpp

```diff
@@ -0,0 +1,655 @@
+#include <gtest/gtest.h>
+#include <sqlext.h>
+#include "../src/drivers.h"
+#include "../src/odbc_handler.h"
+#include "../src/query_generator.h"
+#include "psqlodbc_tests_common.h"
```

Comment:
Removed all the `#include` statements (except for `#include "psqlodbc_tests_common.h"`) & the `using std::pair;` statement.